en/docs/learn/integration-with-bank.md

```diff
@@ -0,0 +1,452 @@
+            "amount":"123",
+            "currency":"EUR"
+         },
+         "cardNumber":"12345678901234",
+         "account":{
+            "iban":"DE98765432109876543210"
+         }
+      }
+   },
+   "consentType":"funds-confirmations",
+   "receipt":{
+      "registrationInformation":"Your contract Number 1234 with MyMerchant is completed with the registration with your bank.",
+      "cardInformation":"My Merchant Loyalty Card",
+      "account":{
+         "iban":"DE98765432109876543210"
+      },
+      "cardNumber":"1234567891234",
+      "cardExpiryDate":"2025-12-31"
+   },
+   "consentFrequency":0
+}
+```
+
+In the core banking system, the required validations should be performed and then the response will be built according 
+to the requirements of the Open Banking Confirmation of Funds specification.
+
+## Configuring sequence files
+
+WSO2 Open Banking API Manager contains a mock bank back end, which is configured in the In sequences by default.
+Therefore, in order to integrate the toolkit with the banking system the In sequence files for Accounts, Payment, and
+CoF must be updated individually.
+
+- They are available in the `<APIM_HOME>/repository/resources/finance/apis/berlin.group.org/<API_NAME>` directory.
+- Replace the `<WSO2_OB_KM_HOST>` placeholder with the hostname of the Identity Server.
+- Replace the `<WSO2_OB_APIM_HOST>` placeholder with the hostname of the API Manager.
+- Replace the following in the sequence file with the core banking system’s API endpoints for production/sandbox environments:
+    - For Accounts (card-accounts) and Payments:
+        1. Open the `<APIM_TOOLKIT_HOME>/repository/resources/apis/berlin.group.org/PSD2API_1.3.6/dynamic-endpoint-insequence-1.3.6` file.
+        2. Replace the `https://localhost:9443/api/openbanking/berlin/backend/services/accounts` URL.
+    - For CoF submission request:
+        1. Open the `<APIM_TOOLKIT_HOME>/repository/resources/apis/berlin.group.org/ConfirmationOfFunds/cof-consents-dynamic-endpoint-sequence-1.3.6` file.
+        2. Replace the `https://localhost:9443/api/openbanking/berlin/backend/services/funds-confirmations` URL.
+
+
+## Accounts Retrieval Endpoints
+
+In some banks, some PSUs may have a certain number of accounts, but not all accounts have the ability to be shared
+externally or to make a payment online. Therefore, in a bank, the shareable account list and the payable account list can
+either be the same or different.
+
+In the default WSO2 Open Banking Berlin Toolkit, at least two APIs are expected to return shareable and payable accounts.
+When passing the `user_id` a JSON response must be returned. Then it automatically loads the accounts list on the consent page.
+
+### Payable Accounts API
+
+The back-end endpoint for the Payable Accounts API is used to retrieve the payable accounts of the user during the
+authentication flow. If a TPP has not provided any debtor account in the payment initiation request, the PSU is able to
```

Comment:
The Berlin specification mandates the debtor account in the initiation requests of all payment types. So this sentence can be removed since it is not needed to mention.

Reply:
Fixed in fe854904d51164a206b027218dd1426b22b49e5b